src/Razor/src/Microsoft.CodeAnalysis.Razor.Workspaces/LSPEditorFeatureDetector.cs

```diff
@@ -7,8 +7,13 @@ internal abstract class LSPEditorFeatureDetector
     {
         public abstract bool IsLSPEditorAvailable(string documentMoniker, object hierarchy);
 
+        /// <summary>
+        /// A remote client is a LiveShare guest or a "VS Cloud Connected Client"
```

Comment:
"VS Cloud Connected Client" was the documentation for `IsVSRemoteClient()`. To confirm, that's Nexus?

Reply:
yup